### PR TITLE
MOL-41: Apple Pay Direct with logged in Shop Account

### PR DESCRIPTION
**Pull request by boxblinkracer**
if the user is already signed in, apple pay direct should use this user instead of creating a guest account.

the user data by apple pay is now stored in the session.

if required, a guest account is being created.

however in both cases, it will be ensured that the order will get the data from apple pay

### CODE REVIEW

**Comment by boxblinkracer** [comment]
on hold (after new release) @barbieswimcrew

**Review by barbieswimcrew — APPROVED** [review]

**Comment by barbieswimcrew** [comment]
@boxblinkracer approved, please merge on your own if the right time has come ;-)

**Comment by boxblinkracer** [comment]
bedankt voor alles @barbieswimcrew  ;)

**Pull request merged**
